TwoFAS/TwoFAS/Other/Generated/T.generated.swift

```diff
@@ -388,8 +388,6 @@ internal enum T {
     internal static func newServices(_ p1: Int) -> String {
       return T.tr("Localizable", "backup__new_services", p1, fallback: "%d new services")
     }
-    /// This file is in a newer format version than the one the app supports
-    internal static let newerFormatNotSupported = T.tr("Localizable", "backup__newer_format_not_supported", fallback: "This file is in a newer format version than the one the app supports")
```

Comment:
This line will actually return as this is generated automatically from translations. But that's OK.